package-lock.json

```diff
@@ -0,0 +1,4473 @@
+{
```

Comment:
We need to completely remove this file, it is required by npm and we don't use it.

Reply:
ok